lib/translations/en.json

```
@@ -0,0 +1,50 @@
+{
+  "common": {
+    "back": "back",
+    "cancel": "Cancel",
+    "caption": "Caption",
+    "embed": "Embed",
+    "save-changes": "Save Changes",
+    "unlink": "Unlink",
+    "video-url": "Video URL:"
+  },
+  "attachments": {
+    "actions-blocked": "You are not permitted to update or delete attachments",
+    "add": "Add Attachment",
+    "delete-confirmation": "Are you sure you want to remove {{entity}}?",
+    "delete-title": "Remove Attachment?",
+    "max-files": "You can only attach {{entity}} files",
+    "name-empty": "Filename cannot be empty",
+    "one-attachment-allowed": "Only one attachment is allowed"
+  },
+  "error": {
+    "invalid-url": "Please enter a valid URL"
+  },
+  "menu": {
+    "attachments": "Attachments",
+    "block-quote": "Block Quote",
```

Let's follow the sentence case for all our labels.
```suggestion
    "block-quote": "Block quote",
```

Ref: https://neeto-engineering.neetokb.com/articles/capitalization